modules/analytics/src/views/full/index.tsx

```diff
@@ -355,7 +353,13 @@ const Analytics: FC<any> = ({ bp }) => {
 
             <Popover>
               <Button icon="calendar">{lang('analytics.dateRange')}</Button>
-              <DateRangePicker onChange={handleDateChange} maxDate={new Date()} value={state.dateRange} />
+              <DateRangePicker
+                locale={langLocale()}
+                localeUtils={LocaleUtils}
+                onChange={handleDateChange}
+                maxDate={new Date()}
+                value={state.dateRange}
+              />
```

Comment:
Per the [blueprintjs documentation](https://blueprintjs.com/docs/#datetime/datepicker.localization) this should translate the datepicker. It does not work and I haven't been able to find a way to make it work. Date translations work just fine but this specific menu item isn't being translated.